theworkoutinfo/ViewControllers/ExerciseDetailViewController.swift

```diff
@@ -38,6 +39,7 @@ class ExerciseDetailViewController: UIViewController {
     }
     
     func setupUI() {
+        let exercise = viewModel.exercise.value
```

Comment:
this should not be done in the setupUI method. it should be done in setupBindings